### ISSUE TITLE
Accessibility: Update roles for drag and drop lists 

### ISSUE THREAD

**Issue by lkopacz** [atlassian/react-beautiful-dnd#2164]
Thanks so much for thinking about accessibility for drag and drop! It's not an easy problem to solve, and I've seen very few folks actually pay attention to it. 

I didn't see any duplicates from a preliminary search, but I think there's some improvement that could be done with roles. For the [Accessibility - Single Option](https://react-beautiful-dnd.netlify.app/?path=/story/accessibility--single-list) story, I noticed that the roles for the draggable list options are buttons. I don't believe the button role gives us the semantic value that we are looking for:

Possible user stories:
- As a keyboard user, I would like to be able to use the arrow keys to navigate through the options when no options are grabbed, not the tab key
- As a screen reader user, I would like to know how many options are in the list and my position in that list (1 of 3)

A few suggestions to make that possible:
- Change the role of the button on the draggable item to be `role="option"`, this accounted for reading the position in the list. (As a separate issue, I think we should make the  instructions shorter, because on VO it doesn't read "1 of 3" until after all of that has been read but that can be thought on more)
- Change `tabIndex` from 0 to -1 to take it out of the taborder, but allow the arrows to navigate through using the arrow keys when the item isn't grabbed.

**Comment by Fedeorlandau** [comment]
I agree the preset instructions are very lengthy. Thanks for raising this.
